jobs/qustodio_backend_barcelona_contract_1.html

```diff
@@ -0,0 +1,39 @@
+---
+title: Backend Engineer
+company: Qustodio
+url: https://www.qustodio.com/en/careers/job/backend-engineer/
+location: Barcelona, Spain
+contract: permanent
+contact:
+    name: Andreas Papaspyropoulos
+    email: ap@qustodio.com
+created: !!timestamp '2017-02-08' 
+tags:
+  - spain
+  - barcelona
+  - python
+  - django
+---
+
+# Overview
+
+Qustodio is a parental control company with offices in the center of Barcelona. 
+We are a multi-national group with a passion for security, software and technology. 
+Our product is a multi-platform device protection solution that is used and loved by thousands of parents, schools and SMBs worldwide.
+
+We're looking for a Python expert to join a great team of Backend and Ops engineers and help us build our backend infrastructure, improve our tools and scale our app for hundreds of thousands of users.
+
```

Comment:
Just a suggestion - if your company offers special benefits this would be a good time to list it. You can include unusual benefits (e.g. flexible working hours, exceptional location, perks)